criu/include/pagemap_scan.h

```diff
@@ -0,0 +1,66 @@
+#ifndef __CR_PAGEMAP_SCAN_H__
+#define __CR_PAGEMAP_SCAN_H__
+
+#include <sys/types.h>
+#include "int.h"
+
```

Comment:
Do you need something to handle if the kernel headers are already defining this? Something like `#ifndef PAGEMAP_SCAN` ?